spark/src/main/java/org/opensearch/sql/spark/asyncquery/model/SparkSubmitParameters.java

```diff
@@ -85,8 +86,13 @@ public Builder clusterName(String clusterName) {
       return this;
     }
 
+    /**
+     * For query in spark submit parameters to be parsed correctly, escape the characters in the
+     * query, then wrap the query with double quotes.
+     */
     public Builder query(String query) {
-      String wrappedQuery = "\"" + query + "\""; // Wrap the query with double quotes
+      String escapedQuery = StringEscapeUtils.escapeJava(query);
```

Comment:
Don't we have this in commons lang library? Is it deprecated?

Reply:
StringEscapeUtils in lang3 is deprecated